docker-compose.yml

```
@@ -18,4 +15,13 @@ services:
       KEYCLOAK_ADMIN_PASSWORD: 'admin'
     ports:
       - '8010:8080'
-    command: start-dev
\ No newline at end of file
+    command: start-dev
+  rabbitmq:
+    image: rabbitmq:latest
```

I'd suggest replacing this direct image reference with a reference to the Dockerfile you included in the repository:

```suggestion
    build: 
      context: ./
      dockerfile: RabbitMqDockerfile
```

---

Oh good call, I didn't know you could do that!